### PR TITLE
new fields addition in BGP counters and BGP prefixes

### PR DESCRIPTION
**Pull request by biplamal**
BGP counters:
in-end-of-rib

BGP-Prefixes:
local-preference
multi-exit-discriminator

### CODE REVIEW

File: models/bgp/open-traffic-generator-bgp.yang [hunk 1]
```diff
@@ -207,6 +207,13 @@ module open-traffic-generator-bgp {
       description
         "The total number of NOTIFICATION messages sent.";
     }
+
+    leaf in-end-of-rib {
+      type otg-types:counter64;
+      description
+        "Number of End-of-RIB markers received indicating the completion of the initial routing update for a particular <AFI, SAFI> address family after the session is established. For the IPv4 unicast address family, the End-of-RIB marker is an UPDATE message with the minimum length. For any other address family, it is an UPDATE message that contains only the MP_UNREACH_NLRI attribute with no withdrawn routes for that <AFI, SAFI>.";
```
Comment (apratimmukherjee) [inline models/bgp/open-traffic-generator-bgp.yang:214]: "The total number of "  seems to be more in sync with previous descriptions.

File: do.sh [hunk 2]
```diff
@@ -23,7 +23,7 @@ install_deps() {
     && apt-get -y install --no-install-recommends apt-utils dialog 2>&1 \
     && apt-get -y install curl git python-is-python3 python3-pip \
     && get_go \
-    && python -m pip install -r requirements.txt \
+    && python -m pip install --prefer-binary -r requirements.txt \
```
Comment (apratimmukherjee) [inline do.sh:26]: Assuming change in intended.

File: Makefile [hunk 3]
```diff
@@ -48,7 +48,7 @@ generate: lint ## go generation target
 		-schema_struct_path=github.com/open-traffic-generator/ixia-c-gnmi-server/pkg/telemetry \
 		-path_struct_package_suffix="" \
 		-base_import_path=github.com/open-traffic-generator/ixia-c-gnmi-server/pkg/telemetry \
-		-path_structs_split_files_count=3 \
+		-path_structs_split_files_count=2 \
```
Comment (apratimmukherjee) [inline Makefile:51]: Assuming changes in this file are all intended.

**Review by apratimmukherjee — APPROVED** [review]
Few minor comments and clarifications on whether some of the changes are intended or not.

File: do.sh (hunk 2 above)
Comment (biplamal) [reply do.sh:26]: Yes, there was bug so fixed it.

File: models/bgp/open-traffic-generator-bgp.yang (hunk 1 above)
Comment (biplamal) [reply models/bgp/open-traffic-generator-bgp.yang]: Done

**Pull request merged**